images/linux/scripts/installers/google-chrome.sh

```diff
@@ -25,3 +25,51 @@ fi
 # Document what was added to the image
 echo "Lastly, documenting what we added to the metadata file"
 DocumentInstalledItem "Google Chrome ($(google-chrome --version))"
+
+CHROME_VERSION=$(google-chrome --version | grep -Eo "([0-9]+\.?){4}" | cut -d "." -f 1,2,3)
+echo "Current Google Chrome version: $CHROME_VERSION"
+
+# Determine latest release of chromedriver
+# Compatibility of Google Chrome and Chromedriver: https://sites.google.com/a/chromium.org/chromedriver/downloads/version-selection
+LATEST_CHROMEDRIVER_VERSION=$(curl "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_$CHROME_VERSION")
+
+# Download and unpack latest release of chromedriver
+echo "Downloading chromedriver v$LATEST_CHROMEDRIVER_VERSION..."
+wget "https://chromedriver.storage.googleapis.com/$LATEST_CHROMEDRIVER_VERSION/chromedriver_linux64.zip"
+unzip chromedriver_linux64.zip
+rm chromedriver_linux64.zip
+mv "chromedriver" "/usr/bin/chromedriver"
+chown root:root /usr/bin/chromedriver
+chmod +x /usr/bin/chromedriver
```

Comment:
maybe move `/usr/bin/chromedriver` to variable like `CHROMEDRIVER_BIN` and use it instead of 3 hard-coded path?

Reply:
Makes sense for me, thanks